libshaderc_spvc/include/spvc/spvc.hpp

```diff
@@ -18,6 +18,7 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <functional>
```

Comment:
nit: I don't think this is needed anymore, since you are no longer using std::function